src/jaxsim/api/contact.py

```diff
@@ -199,16 +199,22 @@ def collidable_point_dynamics(
     )
 
     # Compute the transforms of the implicit frames `C[L] = (W_p_C, [L])`
-    # associated to each collidable point.
+    # associated to the enabled collidable point.
     # In inertial-fixed representation, the computation of these transforms
     # is not necessary and the conversion below becomes a no-op.
+
+    # Get the indices of the enabled collidable points.
+    indices_of_enabled_collidable_points = (
+        model.kin_dyn_parameters.contact_parameters.indices_of_enabled_collidable_points
+    )
```

Comment:
Uhm note that the `indices_of_enabled_collidable_points` property returns a Numpy array, not a tuple: https://github.com/ami-iit/jaxsim/blob/ae908f1a426c6062719eb7f2e8abca4b805ce2e5/src/jaxsim/api/kin_dyn_parameters.py#L761-L763

And I think that a Jax Numpy array can be indexed with a normal numpy array. Do you agree?

Reply:
My bad, I haven't noticed that. Nevertheless, if used inside JIT-ted function, it is converted to a `DynamicJaxprTracer', so probably it would make more sense to just return `jnp.array` inside the property. @diegoferigo WDYT?

Reply:
It becomes a tracer only if converted to a JAX array, not if it's converted to a plain NumPy array. I think that in this case we want to avoid converting it to JAX.

Reply:
Great, thank you both!